src/govuk/components/header/_index.scss

```diff
@@ -142,6 +142,9 @@
     }
   }
 
+  // The govuk-header__link--service-name class is deprecated - use
+  // govuk-header__service-name instead.
+  .govuk-header__service-name,
```

Comment:
Are we happy with this approach? If so, will need to create an issue to remove the `govuk-header__link--service-name` class and attach it to the milestone for v5.0.

The alternative I think is to either:

- live with having a link modifier class on something that isn't a link (which feels icky)
- keep both classes around forever

Reply:
I've created an issue to remove the deprecated class here: https://github.com/alphagov/govuk-frontend/issues/2636